src/sentry/search/snuba/executors.py

```diff
@@ -247,13 +288,35 @@ def snuba_search(
             sample=1,  # Don't use clickhouse sampling, even when in turbo mode.
             condition_resolver=snuba.get_snuba_column_name,
         )
-        rows = snuba_results["data"]
-        total = snuba_results["totals"]["total"]
+        # TODO: need to filter out and/or map the error-specific parameters in the following parameters to the
+        #       transaction-specific ones. CEO: which parameters?
```

Comment:
That's something we will need to figure out. We could talk to discover team about this a bit.

One option here as well could be to use `Dataset.Discover` for both queries, and make one query with `event.type:transaction` and the other with `!event.type:transaction`.

I think discover has already done a lot of the work for filtering across the two datasets, so it's something we could look into.

Reply:
Would `event.type:transaction` be added to the `conditions`? I haven't been able to figure out how/where to add it to the existing condition.

Reply:
Sorry, was using our query language shorthand. 
- `event.type:transaction` would map to `SearchFilter(SearchKey(name="event.type"), "=", SearchValue(raw_value="transaction"))`
- `!event.type:transaction` would map to `SearchFilter(key=SearchKey(name='event.type'), operator='!=', value=SearchValue(raw_value='transaction'))`

We could `convert_search_filter_to_snuba_query` to convert these into conditions and add them to the condition list before querying (or maybe just figure out what the conditions look like after running that and add them in directly)